webview/src/plots/components/App.test.tsx

```diff
@@ -515,21 +471,22 @@ describe('App', () => {
     ).not.toBeInTheDocument()
   })
 
-  it('should not toggle the checkpoint plots section when its header is clicked and its title is selected', async () => {
+  it('should not toggle the custom plots section when its header is clicked and its title is selected', async () => {
     renderAppWithOptionalData({
-      checkpoint: checkpointPlotsFixture
+      comparison: comparisonTableFixture,
+      custom: customPlotsFixture
```

Comment:
`custom` won't render currently without additional data. Planning to changing that and will adjust tests according.